src/modules/home/blogs/blogCard.tsx

```diff
@@ -0,0 +1,50 @@
+import Image from "next/image";
+import { BlogCardProps } from "./helper";
+
+export const Blog = ({
+  name,
+  date,
+  writer_image,
+  title,
+  description,
+  type,
+  image,
+}: BlogCardProps) => {
+  return (
+    <div
+      className="flex items-center justify-center rounded-lg border border-black bg-gray-100 p-2"
+      key={title}
+    >
+      <div className="mx-auto w-full rounded-lg">
+        <Image
+          className="h-full w-full cursor-pointer rounded-lg object-cover transition-all duration-500 hover:scale-95"
+          src={`https://picsum.photos/id/${Math.floor(Math.random())}/200/150`}
```

Comment:
good for now, but please make sure it does not throw error if it can't find the url

Reply:
In future, we will have images from backend, so won't be any issue